src/index.ts

```diff
@@ -290,11 +302,13 @@ export class AcrolinxEndpoint {
     return responsePromise.features;
   }
 
-  public getCheckingCapabilities(accessToken: AccessToken): Promise<CheckingCapabilities> {
+  public async getCheckingCapabilities(accessToken: AccessToken): Promise<CheckingCapabilities> {
```

Comment:
Why the `async` is needed here if there is no await within this function (and return type is Promise)?

Reply:
Removed . (I was fiddling with telemetry stuff inside that function)